app/views/layouts/application.html.erb

```diff
@@ -15,7 +15,7 @@
       <%= link_to "All Items", "/items"%>
       <% if current_user %>
         <%= link_to "Log Out", "/logout", method: :delete %>
-        <p>Logged in as: <%= link_to "#{current_user.name}", "/profile" %></p>
+        <%= link_to "Logged in as: #{current_user.name}", "/profile" %>
```

Comment:
This didn't break any of the previous tests? I'm thinking about the users/navgation_spec.rb and user_logout_spec.rb. Does the 'have_content' still test the link?

Reply:
Good question. Checked before pushing and submitting PR – nothing broken.

Reply:
Made the necessary changes in the nav spec, and the the test still passes. I'll flag this change to be included in a future merge. Thanks for pointing this out.